packages/openid4vp/src/index.ts

```diff
@@ -0,0 +1,21 @@
+export { ClientIdScheme } from './client-identifier-scheme/v-client-id-scheme'
+export { verifyJarmAuthResponse } from './jarm/jarm-auth-response/verify-jarm-auth-response'
+export { JarmClientMetadata } from './jarm/metadata/v-jarm-dcr-metadata'
+export { createOpenid4vpAuthorizationRequest } from './openid4vp-auth-request/create-openid4vp-auth-request'
```

Comment:
since we're already in the openid4vp package, adding openid4vp to the file name maybe is not needed?